conf/core/ConfigOptions.asciidoc

```diff
@@ -3932,10 +3975,12 @@ way snapping.
 === snap.unconnected.ways.snap.way.status
 
 * Data Type: string
-* Default Value: ``
+* Default Value: `Input2`
 
-Status of ways being snapped.  Valid values are 'Input1' or 'Input2'.  Specify 'Input1' to
-snap reference ways to secondary ways or 'Input2' to snap secondary ways to reference ways.
+Status of ways being snapped.  Valid values are 'Input1', 'Input2', or 'Conflated'.  Specify
+'Input1' to snap reference ways to secondary ways or 'Input2' to snap secondary ways to reference
+ways. You may add multiple statuses and separate them with a semicolon. For example, to snap both
+reference and conflated ways use: 'Input1;Conflated'.
 
```

Comment:
`UnconnectedWaySnapper` now supports multiple snap to/from statuses, and I just went ahead and made the default values the way they would commonly be used, snapping secondary to reference.